src/index.html

```diff
@@ -9,9 +9,56 @@
   <link rel="manifest" href="manifest.json">
   <link rel="stylesheet" href="css/main.css">
   <script src="js/phaser.min.js"></script>
+  <style>
+  #overlay {
+    position: fixed;
+    display: none;
+    width: 100%;
+    height: 8%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0.5);
+    z-index: 2;
+    cursor: pointer;
+}
+#text{
+    position: absolute;
+    top: 60%;
+    left: 50%;
+    font-size: 14px;
+    color: white;
+    transform: translate(-50%,-50%);
+    -ms-transform: translate(-50%,-50%);
+}
+  </style>
 </head>
 <body>
+   <div id="overlay" onclick="on()">
+   <div id="text">Arrow keys rotate character. Space to play Attack Animation. 'F' for death Animation
+      Click to move! Press 'E' to hide this window.
+   </div>
+   </div>
+   <div style="padding:20px">
+   </div>
   <script>
+    //a function for listening to keypresses
```

Comment:
This is getting a bit messy, but it's all temporary so we can leave it as is for now. I think informing the users of how to interact with the game is more valuable now than defer it until it's cleaned up.